sdk/schemaregistry/schema-registry/package.json

```diff
@@ -19,17 +19,17 @@
     "execute:samples": "dev-tool samples run samples-dev",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}*.spec.js\"",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-ts-input -- --timeout 5000000 'test/**/*.spec.ts'",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json api-extractor.json src test --ext .ts",
     "pack": "npm pack 2>&1",
     "test:browser": "npm run build:test && npm run integration-test:browser",
     "test:node": "npm run build:test && npm run integration-test:node",
     "test": "npm run build:test && npm run unit-test",
-    "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha -r esm -r ts-node/register --timeout 50000 --reporter ../../../common/tools/mocha-multi-reporter.js --colors --exclude \"test/**/*.browser.ts\" \"test/**/*.ts\"",
+    "unit-test:browser": "npm run integration-test:browser",
```

Comment:
It feels weird that a unit test script is calling an integration test command.

Reply:
fine if they both are running the same command?

Reply:
integration tests are running the same suite but with coverage enabled, notice the `nyc` part in the old integration test script. With unit tests, we do not want to run `nyc` because it gets in the way of the debugger and it is unnecessary overhead.

I am also surprised that the integration scripts do not run the integration command!

Reply:
updated

Reply:
Thanks! I currently see the following:
```
"integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'"
"unit-test:node":            "dev-tool run test:node-ts-input -- --timeout 5000000 'test/**/*.spec.ts'"
```
which makes me wonder, should not `integration-test:node` be running an integration test command that runs `nyc`? I do not expect `integration-test:node` and `unit-test:node` scripts to look similar like this. Same comment applies to the browser scripts as well.

Reply:
test:node-js-input does have nyc

Reply:
![image](https://user-images.githubusercontent.com/97020683/148441459-3fda2961-e007-4c79-853f-290a6ac2a0e7.png)

Reply:
Oh I see, did not notice the difference between `node-ts-input` and `node-js-input`. Thanks for explaining!